src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -158,9 +283,44 @@ const GroupCheckbox = styled(Box)`
   }
 `;
 
-const StyledCount = styled(Count)`
+const StyledPrimaryCount = styled(Count)`
   font-size: 18px;
-  color: ${p => p.theme.gray600};
+  color: ${p => p.theme.gray700};
+`;
+
+const StyledSecondaryCount = styled(Count)`
+  font-size: 18px;
+  color: ${p => p.theme.gray500};
+`;
+
+const TooltipContent = styled(p => (
+  <table {...p}>
+    <tbody>{p.children}</tbody>
+  </table>
+))`
+  margin: 0;
+`;
+
+const TooltipRow = styled('tr')`
+  padding: 4px 8px;
+  display: block;
+`;
+
+const TooltipText = styled('td')`
+  font-weight: normal;
+`;
+
+const TooltipCount = styled('td')`
+  padding-left: 10px;
+  font-weight: bold;
+`;
+
+const StyledIconTelescope = styled(p => (
+  <td {...p}>
+    <IconTelescope size="xs" color={p.color} />
+  </td>
+))`
+  padding-left: 10px;
 `;
 
-export default StreamGroup;
+export default withGlobalSelection(withOrganization(StreamGroup));
```

Comment:
cc @priscilawebdev as this may complicate what you're working on

Reply:
Another question here, is lightweight org still a thing? Or is withOrganization now using the lightweight version by default? @billyvg ?

Reply:
`withOrganization` is "weight agnostic" if you will, it should not be a concern of the components that use it. The thing to be aware of is if you access projects/teams from organization, it will work but we want to migrate away from that pattern.